components/ws-proxy/pkg/proxy/pass.go

```diff
@@ -160,6 +160,7 @@ func createDefaultTransport(config *TransportConfig) *http.Transport {
 		}).DialContext,
 		ForceAttemptHTTP2:     true,
 		MaxIdleConns:          config.MaxIdleConns,                   // default: 100
```

Comment:
@geropl Waht is default in comment? Is it golang default or ours?

Reply:
Goland std lib default.